ccloud-build-app.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-MISSING_CCLOUD_MESSAGE="ccloud is not found. Install Confluent Cloud CLI (https://docs.confluent.io/ccloud-cli/current/install.html) and try again"
+MISSING_CCLOUD_MESSAGE="confluent is not found. Install Confluent CLI (https://docs.confluent.io/ccloud-cli/current/install.html) and try again"
```

Comment:
@bbejeck this might be the preferred docs link (CLI v2): https://docs.confluent.io/confluent-cli/current/install.html

Reply:
Also, do you want to rename `MISSING_CCLOUD_MESSAGE`?